ipinfo/cmd_tool_prefix_addr.go

```diff
@@ -21,7 +21,7 @@ func printHelpToolPrefixAddr() {
 		`Usage: %s tool prefix addr <cidr>
 
 Description:
-	returns prefix's IP address.
+ returns IP address of CIDR using prefix.
```

Comment:
Using wrong number of spaces

Also let us make the description just "returns the base IP address of a prefix."

Reply:
ok sure